### PR TITLE
Create faulthandler.travis.yml

### PR DESCRIPTION
**Pull request by gururajrkatti**

### CODE REVIEW

File: travis-ymls/faulthandler.travis.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+python:
+  - 2.7
+arch:
+  - amd64
+  - ppc64le
+install: pip install tox-travis
+script: tox
+notifications:
+  email:
+    - victor.stinner@gmail.com
```
Comment (seth-priya) [inline travis-ymls/faulthandler.travis.yml:22]: does this actually send out the notifications to the given email id for all builds even for forked branches?

**Comment by gururajrkatti** [comment]
@seth-priya did you get a chance to look into this ?

**Comment by seth-priya** [comment]
I don't see a response yet on the above question that I had for you @gururajrkatti , am I missing something?

**Comment by gururajrkatti** [comment]
@seth-priya Had sent the below response
====================
does this actually send out the notifications to the given email id for all builds even for forked branches?

Hi @seth-priya , below is what I saw in the travis documentation Pull Request builds do not trigger email notifications. I think it means forked branch build notifications won't be sent ?
=================

**Pull request merged**